tests/run-make/thumb-none-qemu/rmake.rs

```diff
@@ -14,49 +14,40 @@
 //!
 //! FIXME: https://github.com/rust-lang/rust/issues/128733 this test uses external
 //! dependencies, and needs an active internet connection
-//!
-//! FIXME: https://github.com/rust-lang/rust/issues/128734 extract bootstrap cargo
-//! to a proper command
 
 //@ only-thumb
 
 use std::path::PathBuf;
 
-use run_make_support::{cmd, env_var, path_helpers, target};
+use run_make_support::{cargo, cmd, env_var, path, target};
 
 const CRATE: &str = "example";
 
 fn main() {
     std::env::set_current_dir(CRATE).unwrap();
 
-    let bootstrap_cargo = env_var("BOOTSTRAP_CARGO");
-    let path = env_var("PATH");
+    let path_env = env_var("PATH");
     let rustc = env_var("RUSTC");
-
-    let target_dir = path_helpers::path("target");
-    let manifest_path = path_helpers::path("Cargo.toml");
-
-    let debug = {
-        let mut cmd = cmd(&bootstrap_cargo);
-        cmd.args(&["run", "--target", &target()])
-            .env("RUSTFLAGS", "-C linker=arm-none-eabi-ld -C link-arg=-Tlink.x")
-            .env("CARGO_TARGET_DIR", &target_dir)
-            .env("PATH", &path)
-            .env("RUSTC", &rustc);
-        cmd.run()
-    };
-
-    debug.assert_stdout_contains("x = 42");
-
-    let release = {
-        let mut cmd = cmd(&bootstrap_cargo);
-        cmd.args(&["run", "--release", "--target", &target()])
-            .env("RUSTFLAGS", "-C linker=arm-none-eabi-ld -C link-arg=-Tlink.x")
-            .env("CARGO_TARGET_DIR", &target_dir)
-            .env("PATH", &path)
-            .env("RUSTC", &rustc);
-        cmd.run()
-    };
-
-    release.assert_stdout_contains("x = 42");
+    let target_dir = path("target");
+    let manifest_path = path("Cargo.toml");
+
+    // Debug
+    cargo()
+        .args(&["run", "--target", &target()])
+        .env("RUSTFLAGS", "-C linker=arm-none-eabi-ld -C link-arg=-Tlink.x")
+        .env("CARGO_TARGET_DIR", &target_dir)
+        .env("PATH", &path_env)
```

Comment:
Now that I see it again, `PATH` and `RUSTC` should be inherited by default, right?

Reply:
Hm, I'm not 100% sure...

Reply:
Unless we do some magic inside of run-make-support, creating a Rust `Command` should automatically propagate env. vars, if https://doc.rust-lang.org/std/process/struct.Command.html#method.env_clear is not used.

Reply:
Actually no I think you're right, since `rmake.rs` is given `RUSTC` and `PATH` (at least on windows) which *should* mean that `cargo` as a `Command` also inherits `RUSTC` and `PATH`? I suppose it doesn't hurt to be explicit in *this* case...

Reply:
Sure, let's keep it explicit. You can r=me once CI is green.

Reply:
Good point, I'll remove it

Reply:
Uhh the comment timings are slightly awkward lol, I removed the unnnecessary `PATH` and `RUSTC` since they should be inherited by `rmake.rs`.